utils/inbucket.go

```diff
@@ -37,6 +38,12 @@ type JSONMessageInbucket struct {
 		Text string
 		HTML string `json:"Html"`
 	}
+	Attachments []struct {
+		Filename     string
+		ContentType  string `json:"content-type"`
+		DownloadLink string `json:"download-link"`
+		Bytes        []byte `json:"-"`
+	}
```

Comment:
Attachments are now supported for InBucket tests. This code will automatically download attachments and include them in the returned objects, so don't write a unit test that emails gigabyte sized attachments :)